pylint/lint.py

```diff
@@ -1843,6 +1861,14 @@ def cb_verbose_mode(self, *args, **kwargs):
         self.verbose = True
 
 
+def cb_list_extensions(option, optname, value, parser):
+    """List all the extensions under pylint.extensions"""
+    for extension in extensions.extension_names:
+        print(extension)
+        print(extensions.get_extension_documentation(extension))
```

Comment:
This command should just list the extensions without the documentation. The documentation can be accessed on the documentation website or we can add a separate command to grab the entire documentation of a plugin, but right now I suggest to remove this line.